docker/Dockerfile

```diff
@@ -12,5 +12,7 @@ WORKDIR /home
 
 ARG branch
 ENV branch ${branch:-v2.1.2}
+ARG slug
+ENV slug ${slug:-dib-lab/khmer}
```

Comment:
This is the bit that makes sure to set `slug` to dib-lab/khmer when `TRAVIS_PULL_REQUEST_SLUG` is empty?

Reply:
Correct!